src/server/graphql/models/User/userMutation.js

```diff
@@ -168,26 +168,37 @@ export default {
 
       // RESOLUTION
       // propagate denormalized changes to TeamMember
-      const dbProfile = await r.table('TeamMember')
+      const dbProfileChanges = await r.table('TeamMember')
         .getAll(id, {index: 'userId'})
         .update(validUpdatedUser)
         .do(() => {
           return r.table('User')
             .get(id)
             .update(validUpdatedUser, {returnChanges: 'always'});
```

Comment:
we should probably update the `updatedAt` field, which would guarantee that we return a change, which means we can return `('changes')(0)('new_val')`. but we can do that after the merge